core/src/test/scala/com/evolutiongaming/catshelper/ResourceBreakFlatMapChainTest.scala

```diff
@@ -18,17 +18,17 @@ class ResourceBreakFlatMapChainTest extends AsyncFunSuite with Matchers {
     val result = ()
       .pure[IO]
       .toResource
-      .multiply(100)
-      .multiply(100)
+      .multiply(1000)
+      .multiply(1000)
       .use { _ => ().pure[IO] }
       .attempt
       .void
 
     FromFuture[IO]
       .apply { result.toFuture }
-      .timeout(1.second)
+      .timeout(3.second)
       .attempt
-      .flatMap { a => IO { a should matchPattern { case Left(_: TimeoutException) => } } }
+      .flatMap { a => IO { a should matchPattern { case Right(()) => } } }
```

Comment:
@sirmax looks like https://github.com/typelevel/cats-effect/issues/469 is fixed now